scripts/docker-compose/docker-compose.linux.override.yml

```diff
@@ -101,52 +101,39 @@ services:
     extra_hosts:
       - "host.docker.internal:host-gateway"
 
-  zgw-referentielijsten-database:
-    user: "${UID}:${GID}"
+  gbamock:
     # Linux workaround for host.docker.internal support
     # see: https://stackoverflow.com/a/67158212
     extra_hosts:
       - "host.docker.internal:host-gateway"
 
-  zgw-referentielijsten.local:
```

Comment:
? I already removed them last week. They are not present on the main branch. Maybe you need to update this branch?

Reply:
That's just for the override for use in Linux with extra_hosts & the correct system user for Docker

Reply:
ah clear!